packages/shopify-cart/README.md

```diff
@@ -61,6 +61,7 @@ The cart client accepts the following parameters:
 - `fetchClient` (`function`) - optional Fetch API-compatible request function
 - `country` (`string`) - optional [Shopify Country Code][shopify-cart-country-code] used for cart queries/responses. Defaults to `ZZ` (unknown region) if not supplied.
 - `language` (`string`) - optional [Shopify Language Code][shopify-cart-language-code] used for cart queries/responses. Defaults to `EN` if not supplied.
+- `locale` (`string`) - optional [Nacelle Locale Code]used to assign `NacelleCartMerchandise` the correct `nacelleEntryId`. Defaults to `en-US` if not supplied.
```

Comment:
@alexcasche this isn't quite right yet

![Screen Shot 2022-09-27 at 11 16 03 AM](https://user-images.githubusercontent.com/5732000/192566196-f7de4761-2a7b-41c2-bbc0-9381baad5aca.png)